site/_docs/installation.md

```diff
@@ -31,16 +32,18 @@ earlier, for CoffeeScript support).
   </p>
 </div>
 
-## Install with RubyGems
+## Install with Bundler and RubyGems
 
-The best way to install Jekyll is via
+The best way to install Jekyll is using a Gemfile via Bundler
 [RubyGems](http://rubygems.org/pages/download). At the terminal prompt,
 simply run the following command to install Jekyll:
 
 {% highlight shell %}
-$ gem install jekyll
+$ gem install bundler
 {% endhighlight %}
 
+Now you can install Jekyll simply by running `bundle install`.
```

Comment:
The Gemfile gets created with `gem "jekyll", "3.2.1"` included even with the latest release, v3.2.1. I'm just taking the generation outside `new.rb` to an erb template.